code-orange/ViewController.swift

```diff
@@ -9,17 +9,50 @@
 import UIKit
 import GoogleMaps
 
-protocol CurrentLocationProvider: class {
+protocol LocationProvider: class {
   var currentLocation: CLLocationCoordinate2D { get }
 }
 
+protocol CoordinatesProviderDelegate: class {
```

Comment:
same here

Reply:
This one belongs with the next one - moved both to a new file!